DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/jtag.sdc

```diff
@@ -0,0 +1,84 @@
+#****************************************************************************
```

Comment:
Same remark: I hope that we can avoid shipping this file

Reply:
this one needs to stay, it's from the GHRD as described in **Board-specific Considerations** in the readme.